arviz/plots/backends/matplotlib/ecdfplot.py

```diff
@@ -0,0 +1,189 @@
+"""Matplotlib ecdfplot."""
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.stats import uniform, binom
+
+from arviz.plots.plot_utils import _scale_fig_size
+from arviz.plots.backends.matplotlib import backend_kwarg_defaults, create_axes_grid, backend_show
+
+
+def plot_ecdf(
+    values,
+    values2,
+    distribution,
+    difference,
+    pit,
+    confidence_bands,
+    granularity,
+    num_trials,
+    alpha,
+    figsize,
+    ecdf_fill,
+    ax,
+    show,
+    backend_kwargs,
+):
+    """Matplotlib ecdfplot."""
+    if backend_kwargs is None:
+        backend_kwargs = {}
+
+    backend_kwargs = {
+        **backend_kwarg_defaults(),
+        **backend_kwargs,
+    }
+
+    (figsize, _, _, _, _, _) = _scale_fig_size(figsize, None)
+    backend_kwargs.setdefault("figsize", figsize)
+    backend_kwargs["squeeze"] = True
+
+    if ax is None:
+        _, ax = create_axes_grid(1, backend_kwargs=backend_kwargs)
+
+    n = len(values)
+
+    if confidence_bands:
+        if pit:
+            x = np.linspace(1 / granularity, 1, granularity)
+            z = x
+            values = (
+                distribution(values)
+                if distribution
+                else compute_ecdf(values2, values) / len(values2)
+            )
+        else:
+            x = np.linspace(values[0], values[-1], granularity)
+            z = distribution(x) if distribution else compute_ecdf(values2, x) / len(values2)
+
+        gamma = get_gamma(n, z, granularity, num_trials, alpha)
+        lower, higher = get_lims(gamma, n, z)
+
+        x_coord, y_coord = [], []
+        if not difference:
+            for _, x_i in enumerate(x):
+                f_x_i = compute_ecdf(values, x_i) / n
+                x_coord.append(x_i)
+                y_coord.append(f_x_i)
+
+            plt.step(x_coord, y_coord, where="post")
+
+            if ecdf_fill:
+                plt.fill_between(x, lower / n, higher / n, color="C0", alpha=0.2)
```

Comment:
Can the color and alpha be set from the `plot_ecdf` function somehow?

Reply:
I will add plot_kwargs, fill_kwargs so that the plots can be customized